src/main/java/rs/teslaris/core/controller/ConferenceController.java

```diff
@@ -26,12 +33,22 @@ public class ConferenceController {
 
     private final ConferenceService conferenceService;
 
+    private final EventService eventService;
+
 
     @GetMapping
     public Page<ConferenceDTO> readAll(Pageable pageable) {
         return conferenceService.readAllConferences(pageable);
     }
 
+    @GetMapping("/simple-search")
+    Page<EventIndex> searchConferences(
+        @RequestParam("tokens")
+        @NotNull(message = "You have to provide a valid search input.") List<String> tokens,
+        Pageable pageable) {
+        return eventService.searchEvents(tokens, pageable, EventType.CONFERENCE);
```

Comment:
ConferenceServiceImpl is extending EventServiceImpl. ConferenceService interface is not declaring this method, which might be the issue, but I would suggest to add declaration of `searchEvents(tokens, pageable)` into ConferenceService, and than implement that just by invoking this line. At the end ConferenceController will only inject ConferenceService (not EventService)

Reply:
I refactored it by just specifying method signature in `ConferenceService` interface, implementation is automatically pulled from parent class. Is that ok?

Reply:
It means you could search other types of events (not just conferences) through ConferenceService. I would make it more strict:

ConferenceService

searchConferences(tokens, pageable)

ConferenceServiceImpl

searchConferences(tokens, pageable){
   return searchEvents(tokens, pageable, EventType.Conference);
}

Reply:
Done